packages/react-magma-dom/src/components/Pagination/Pagination.tsx

```diff
@@ -65,6 +66,16 @@ export interface BasePaginationProps
    * @default false
    */
   showLastButton?: boolean;
+  /**
+   * If true, shows the Pagination pages in a dropdown style.
+   * @default button
+   */
+  type?: PaginationType;
+}
+
+export enum PaginationType {
+  default = 'button',
```

Comment:
I think we should call the types `simple` and `classic` (at least that's what Orion used in the designs)
```suggestion
  classic = 'classic',
```

Reply:
Should this be renamed in the files as well?